.github/workflows/common.yml

```diff
@@ -77,56 +83,56 @@ jobs:
-              exit $exit_code
-            else
-              echo "$out" ${{ env.REDIRECT }}
-            fi
-          else
-            echo "$out"
-          fi
-
-      - name: Run optional checks/linters
-        env:
-          COMMAND: "bin/golangci-lint run --out-format=line-number"
-          REDIRECT: "| bin/reviewdog -f=golangci-lint -reporter=github-pr-review"
-          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.ROBOT_TOKEN || secrets.GITHUB_TOKEN }}
-        run: |
-          if out=$( ${{ env.COMMAND }} ); exit_code=$?; [ $exit_code -ne 0 ]; then
-            if [ $exit_code -gt 1 ] || ${{ github.event.pull_request == null }}; then
-              echo "$out"
-              exit $exit_code
-            else
-              echo "$out" ${{ env.REDIRECT }}
-            fi
-          else
-            echo "$out"
-          fi
+#      - name: Run golangci-lint
+#        env:
+#          COMMAND: "bin/golangci-lint run -c=.golangci-required.yml --out-format=line-number"
+#          REDIRECT: "| bin/reviewdog -f=golangci-lint -reporter=github-pr-review -fail-on-error=true"
+#          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.ROBOT_TOKEN || secrets.GITHUB_TOKEN }}
+#        run: |
+#          if out=$( ${{ env.COMMAND }} ); exit_code=$?; [ $exit_code -ne 0 ]; then
+#            if [ $exit_code -gt 1 ] || ${{ github.event.pull_request == null }}; then
+#              echo "$out"
+#              exit $exit_code
+#            else
+#              echo "$out" ${{ env.REDIRECT }}
+#            fi
+#          else
+#            echo "$out"
+#          fi
+#
+#      - name: Run go-consistent
+#        env:
+#          COMMAND: 'bin/go-consistent -pedantic -exclude "tests" ./...'
+#          REDIRECT: "| bin/reviewdog -f=go-consistent -reporter=github-pr-review -fail-on-error=true"
+#          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.ROBOT_TOKEN || secrets.GITHUB_TOKEN }}
+#        run: |
+#          if out=$( ${{ env.COMMAND }} ); exit_code=$?; [ $exit_code -ne 0 ]; then
+#            if [ $exit_code -gt 1 ] || ${{ github.event.pull_request == null }}; then
+#              echo "$out"
+#              exit $exit_code
+#            else
+#              echo "$out" ${{ env.REDIRECT }}
+#            fi
+#          else
+#            echo "$out"
+#          fi
+#
+#      - name: Run optional checks/linters
+#        env:
+#          COMMAND: "bin/golangci-lint run --out-format=line-number"
+#          REDIRECT: "| bin/reviewdog -f=golangci-lint -reporter=github-pr-review"
+#          REVIEWDOG_GITHUB_API_TOKEN: ${{ secrets.ROBOT_TOKEN || secrets.GITHUB_TOKEN }}
+#        run: |
+#          if out=$( ${{ env.COMMAND }} ); exit_code=$?; [ $exit_code -ne 0 ]; then
+#            if [ $exit_code -gt 1 ] || ${{ github.event.pull_request == null }}; then
+#              echo "$out"
+#              exit $exit_code
+#            else
+#              echo "$out" ${{ env.REDIRECT }}
+#            fi
+#          else
+#            echo "$out"
+#          fi
```

Comment:
why did we comment this?

Reply:
https://github.com/percona/pmm/pull/1040#issuecomment-1229997873